tests/test_cr_detection.py

```diff
@@ -22,21 +22,21 @@ def find_plateaus_iit(streak_row, fwc, sat_thresh, plat_thresh, cosm_filter):
 
     Args:
         streak_row (float):
-            Row with possible cosmics.
-        fwc (float): 
-            Full well capacity of detector *in DNs*.  Note that this may require a
-            conversion as FWCs are usually specified in electrons, but the image
-            is in DNs at this point.
-        sat_thresh (float): 
-            Multiplication factor for fwc that determines saturated cosmic pixels.
-        plat_thresh (float): 
-            Multiplication factor for fwc that determines edges of cosmic plateu.
-        cosm_filter (int): 
-            Minimum length in pixels of cosmic plateus to be identified.
+        Row with possible cosmics.
```

Comment:
this lack of indentation makes it a bit difficult to read. can we indent it like the other docstrings?